src/action/payment.js

```diff
@@ -114,6 +119,20 @@ class PaymentAction {
     }
     await this._transaction.update();
   }
+
+  async getFeeEstimate(destination, satAmt) {
```

Comment:
We should probably rename this to `estimateLightningFee` as we will also have an o-chain `estimateFee`:
https://github.com/lightninglabs/lightning-app/pull/318/files

Also, all actions use paramter destructuring e.g. ({ param }) for all function. This makes it easier to read the code consuming the api and also has some other benefits. See:
http://2ality.com/2015/01/es6-destructuring.html#simulating-named-parameters-in-javascript